account/views.py

```diff
@@ -1,50 +1,86 @@
-# from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# import json
-# from django .views import View
-# from django.http   import JsonResponse
+from django.views import View
+from django.http  import JsonResponse, HttpResponse
 
-# from account.models import Account
+from account.models import Account
+from django.db import IntegrityError
 
-# class CreateView(View):
-#     def post(self,request):
-#         data = json.load(request.body)
-#         Account(
-#             name     = data['name'],
-#             email    = data['email'],
-#             password = data['password'],
-#             language = data['language'],
-#             country  = data['country'],
-#             birthday = data['birthday'],
-#             phone    = data['phone'],
-#         )
-        
-#         if  Account.objects.filter(name = data['name']).exists() == True:
-#             return JsonResponse({"message":"이미 존재하는 아이디입니다."},status = 401)
+class  SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if '@' not in data['email']:
+                return JsonResponse(
+					{"message":"Email_Not_Verified"},
+					status = 404
+                )
```

Comment:
```suggestion
            if '@' not in data['email']:
                return JsonResponse({"message":"Email_Not_Verified"}, status = 404)
```

response가 별로 길지 않기 때문에 한 줄에 작성하셔도 좋습니다 😄 아래 다른 코드들도 모두 한 줄로 바꿔주세요
그리고 status cod 404는 페이지를 찾을 수 없을 때 발생하는 에러이기 때문에 이메일이 잘못되었음을 나타내기에는 적절하지 않습니다. `400`등으로 바꿔주세요

Reply:
수정했습니다